### DISCUSSION TITLE
Learning with Synthesia on iPad

### DISCUSSION THREAD

**Discussion by keuchhusten78** [onlaj/Piano-LED-Visualizer#368] [Q&A] [answered]
Hello,

my Problem is that i can‘t get Synthesia on my iPad to light up the Notes that i have to play. I‘m connect to the Pi via bluetooth.
The learning mode on LED Visualizer works fine, but i like to see witch notes i have to play next.
I testet different Midi Ports on Visualizer, but i can‘t get it to work!
Any Tipps for a noob
Thank you

**Comment by onlaj** [comment, marked as answer]
Hey.
Is Synthesia not working at all or just the key lighting?
Can you show me your Synthesia's settings?

**Reply by keuchhusten78** [reply]
Hey onlaj,

i hooked up synthesia via bluetooth to the Rasbpi.
As input in synthesia i selected the rasbpi, that works fine.
But the lightstrip don‘t show me keys that i have to play, just the keys that i actual hit! 
I attached photos aof the settings, thank you for your help!

Regards Alex



![48D8B044-DA13-4172-AF7B-0458319D289E](https://user-images.githubusercontent.com/109147472/178492436-623550c9-d3ba-403a-98cc-efded5156fa2.png)
![8D0D535A-D6A0-4812-A104-99D7BCF261FB](https://user-images.githubusercontent.com/109147472/178492456-06b05720-6ced-4144-95ca-96d9574ceb95.png)
![B9AD92DD-39DD-429B-ACD1-E52FC540794A](https://user-images.githubusercontent.com/109147472/178492464-2048b92f-768c-4c8b-a6f7-37ea47c715bd.png)

**Reply by onlaj** [reply]
In Visualizer settings set Bluetooth device as your input.

Those are my settings, maybe try to copy them:
![chrome_0wz4PS0M5O](https://user-images.githubusercontent.com/9073428/178495318-0c173296-96ee-4970-84c9-e591602e1e71.png)

"Key lights" are set to "Finger-based"

**Reply by keuchhusten78** [reply]
Thanky You so much onlaj,

now it works perfectly👍🏻 I got my keyboard as input in visualizer, now with the iPad as input it works!

Thx a lot

Regards
Alex

**Reply by GAMERLEGEND90** [reply]
how did you connect the ipad to rasberry via Bluetooth?

**Comment by GAMERLEGEND90** [comment]
> Hello,
> 
> my Problem is that i can‘t get Synthesia on my iPad to light up the Notes that i have to play. I‘m connect to the Pi via bluetooth. The learning mode on LED Visualizer works fine, but i like to see witch notes i have to play next. I testet different Midi Ports on Visualizer, but i can‘t get it to work! Any Tipps for a noob Thank you

how do i connect my ipad to Bluetooth to rasberry?
